### PR TITLE
[Symfony 2.7] Removed deprecation warnings

### PR DESCRIPTION
**Pull request by lolautruche**
* Used configureOptions() instead of setDefaultOptions in form types
* Fixed deprecations in routing.yml
* Removed usage of security context

Tested locally, with unit tests and in browser.

### CODE REVIEW

File: Form/Type/SectionType.php [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
 use Symfony\Component\OptionsResolver\OptionsResolverInterface;
```
Comment (andrerom) [inline Form/Type/SectionType.php]: seems this can be removed now in most cases here

**Comment by andrerom** [comment]
+1

**Comment by dpobel** [comment]
+1

File: Form/Type/SectionType.php (hunk 1 above)
Comment (lolautruche) [inline Form/Type/SectionType.php]: Thanks. Fixed.

**Pull request merged**